cmd/skupper/skupper_kube_site.go

```diff
@@ -167,6 +167,7 @@ func (s *SkupperKubeSite) CreateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&routerCreateOpts.ConfigSync.Memory, "config-sync-memory", "", "Memory request for config-sync pods")
 	cmd.Flags().StringVar(&routerCreateOpts.ConfigSync.CpuLimit, "config-sync-cpu-limit", "", "CPU limit for config-sync pods")
 	cmd.Flags().StringVar(&routerCreateOpts.ConfigSync.MemoryLimit, "config-sync-memory-limit", "", "Memory limit for config-sync pods")
+	cmd.Flags().BoolVarP(&routerCreateOpts.EnableClusterPermissions, "enable-cluster-permissions", "", false, "Enable cluster wide permissions")
```

Comment:
I think this option needs a clearer description. E.g. it could clarify that it gives the service controller cluster wide permissions, or it could say that it allows targets in other namespaces to be exposed.

Reply:
It makes sense, thanks! That's why I asked in the comment above:

> Currently, if a specific site has been previously created and the user tries to enable all cluster permissions later on they won't be able, since the cluster roles are just created, not updated. Would this be an issue?

It's addressed now